paper/paper.md

```diff
@@ -0,0 +1,85 @@
+
+For development, the initial segmentation uses superpixels generated with the SLIC algorithm.  These are computed on whole slide images in a tiled manner so that they can work on arbitrarily large images, and the tile boundaries are properly handled to avoid visible artifacts.  Either of two basic models can be trained and used for predictions: small-scale CNN using image features implemented in tensorflow/keras or torch, or a huggingface foundation model that generates a one-dimensional feature vector.  The certainty criteria for which segments should be labeled next can also be selected, and includes confidence, margin, negative entropy, and the BatchBALD [@batchbald2019] algorithm.
+
+We had a placental pathologist provide feedback to validate the efficiency of the user interface and utility of the process.
+
+# Basic Workflow
+
+When starting a new labeling project, the user selects how superpixels are generated, which certainty metric is used for determining the optimal labeling order, and what features are used for model training.  The labeling mode allows defining project labels and performing initial labeling.  This mode can also be used to add new label categories or combine two categories if they should not have been distinct.
+
+![The Bulk Labeling interface showing one of the project images divided into superpixels with some categories defined.  A user can "paint" areas with known labels as an initial seed for the guided labeling process](../docs/screenshots/initial_labels.png)
+
+Once some segments have been labeled and an initial training process has been performed, additional segments are shown with their predictions.  The user can use keyboard shortcuts or the mouse to confirm or correct labels.  These are presented in an order that maximizes the utility of improving the model based on the originally selected certainty metric.
+
+![The Guided Labeling interface showing a row of superpixels to be labeled and part of a whole slide image](../docs/screenshots/active_learning_view.png)
+
+To check on overall behavior or correct mistakes, there is a review mode that allows seeing all labeled segments with various filtering and sorting options.  This can be used to check agreement between pathologists or determine how well the model agrees with the manually labeled data.
+
+![The Review interface showing labeled superpixels in each category](../docs/screenshots/reviewmode.png)
+
+The whole slide image data in these figures are from data generated by the TCGA Research Network [@TCGA].
```

Comment:
I'd change "TCGA Research Network" to "The Cancer Genome Atlas Program (TCGA)" ... unless there is also a "research network" that I don't know about.

Reply:
I was just following this: https://www.cancer.gov/ccg/research/genome-sequencing/tcga/using-tcga-data/citing